### DISCUSSION TITLE
repository language not detected

### DISCUSSION THREAD

**Discussion by BashMocha** [github-linguist/linguist#5813] [Classification] [answered]
Hi, I am new to GitHub.
I have made a simple snake game in java in my repository.
[link to the repository](https://github.com/CheesyFrappe/Java-Snake-Game).
GitHub is not displaying any language for my repository.

**Comment by lildude** [comment, marked as answer]
Moved this to discussions as recommended on the page you used to open the original issue (the last point 😉 ).

This looks like the same thing as experienced in https://github.com/github/linguist/discussions/5801

You've only made a single commit. Make another and analysis will happen.
